constants.js

```diff
@@ -4,6 +4,9 @@ global.__base = appRoot + '/';
 global.__appbase = appRoot + '/app/';
 global.__tmpbase = appRoot + '/tmp/';
 
+global.logger = require(__appbase + '/services/logger');
+
 global.appConfig = {
-    IS_LOCAL_DB: false
+    IS_LOCAL_DB: false,
+    APP_ENV: process.env.NODE_ENV || 'development'
```

Comment:
Good! This is the correct way of initializing these kind of values! Always have a fallback!!! (aka `|| something`)